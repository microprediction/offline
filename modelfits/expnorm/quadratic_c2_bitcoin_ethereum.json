{"g1": 0.02953156326937898, "g2": 11.130468405640917, "logK": -4.603573565601684, "loc": 0.10198517381335714, "logScale": 0.8967830452238886}