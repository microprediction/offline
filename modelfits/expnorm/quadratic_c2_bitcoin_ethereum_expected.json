{"g1": 0.944424416121515, "g2": 6.446834291688189, "logK": 0.9996780139469582, "loc": -0.1431376075673064, "logScale": -1.7037072456127489}